static/app/views/replays/detail/userActionsNavigator.tsx

```diff
@@ -18,14 +15,48 @@ import {useReplayContext} from 'sentry/components/replays/replayContext';
 import {relativeTimeInMs} from 'sentry/components/replays/utils';
 import {t} from 'sentry/locale';
 import space from 'sentry/styles/space';
-import {Crumb, RawCrumb} from 'sentry/types/breadcrumbs';
+import type {Crumb, RawCrumb} from 'sentry/types/breadcrumbs';
+import {BreadcrumbType} from 'sentry/types/breadcrumbs';
 import {EventTransaction} from 'sentry/types/event';
+import isErrorCrumb from 'sentry/utils/replays/isErrorCrumb';
 
 type Props = {
   crumbs: Array<RawCrumb>;
   event: EventTransaction;
 };
 
+// In the future if we want to show more items at
+// the EventChapter, we should add to this array.
+const USER_ACTIONS = [
+  BreadcrumbType.USER,
+  BreadcrumbType.UI,
+  BreadcrumbType.EXCEPTION,
+  BreadcrumbType.NAVIGATION,
+];
+
+/**
+ * We want to treat the raw crumb of "category:sentry.event,level:error" as an exception when transforming
+ */
+function transformCrumbs(breadcrumbs: Array<RawCrumb>): Crumb[] {
```

Comment:
This would be great in `static/app/components/replays/utils.tsx` or someplace similar. The Timeline has the same issue.

Reply:
I'd prefer it to live close to the component that uses it unless it becomes used by more than 1 component. Case can be made though that we'll want to test it - I think we should keep it in `app/views/replays/utils` then

Reply:
cool

the timeline will be the second place that uses it. currently it's making the same call to the original `transformCrumbs` and `filter()`: https://github.com/getsentry/sentry/blob/bd68084e7c5c6a608e44a5c49876772cb987e88b/static/app/components/replays/breadcrumbs/replayTimeline.tsx#L40-L41